dask_sql/physical/rex/core/call.py

```diff
@@ -52,6 +52,10 @@ def as_timelike(op):
         raise ValueError(f"Don't know how to make {type(op)} timelike")
 
 
+def is_timestamp_nano(obj):
+    return pd.api.types.is_integer_dtype(obj)
```

Comment:
I think my preference here would be to use the `is_integer_dtype` check directly in place of `is_timestamp_nano` since we aren't doing any additional checks on `obj` here